index.js

```diff
@@ -441,6 +441,27 @@ Router.prototype.route = function route (path) {
   return route
 }
 
+/**
+ * List all registered routes.
+ *
+ * @return {Array} An array of route paths
+ * @private
```

Comment:
```suggestion
 * @public
```
This is on the proto so it's public

Reply:
Yep, by "private" I meant the underscore in the function name, but I had some doubts about doing it that way. It's also true that anyone could still access it, so it doesn't really prevent usage. Either way, I'll remove the underscore from the function name and document it as public. Any suggestions for the function name are more than welcome!